node/packages/parser/src/ethereum/yearn.ts

```diff
@@ -0,0 +1,33 @@
+import { Tx } from '@shapeshiftoss/blockbook'
+import { TxSpecific as ParseTxSpecific, YearnTx } from '../types'
+import shapeShiftRouter from './abi/shapeShiftRouter'
+import yearnVault from './abi/yearnVault'
+import { addABI, decodeMethod } from 'abi-decoder'
+import { GenericParser } from './index'
+
+const ROUTER_CONTRACT = '0x6a1e73f12018D8e5f966ce794aa2921941feB17E'
```

Comment:
Is this the ShapeShift router contract address? Would be worth specifying since there are various contracts being interacted with in the yearn ecosystem

Reply:
Indeed. I've improved this in https://github.com/shapeshift/unchained/pull/267/commits/712a59eb5a2e8061d2a7730f4f4a319b19369a21 by using descriptive constants.